react-component-template/index.jsx

```diff
@@ -12,19 +14,8 @@ export default class {{PascalName}} extends Component {
     name: '{{titleName}}'
   }
 
-  constructor (props) {
-    super(props)
-    // initialize state values
-    this.state = {}
-  }
-
-  // use the pure-render mixin without the mixin. This allows us to use es6
-  // classes and avoid "magic" code. NOTE: if this component is used directly
-  // by react-router, you should delete it, otherwise, the <Link> component will
-  // not cause a re-render
-  shouldComponentUpdate (...args) {
-    return shouldComponentUpdate.apply(this, args)
-  }
+  // initialize state values
+  state = {}
```

Comment:
Okay, so I think @Jon-Biz and I are in agreement that any state initialization should be removed. @tedbreen how do you feel about that?